classes/ActionScheduler_AdminView.php

```diff
@@ -228,6 +228,10 @@ public function add_help_tabs() {
 					'<h2>' . sprintf( __( 'About Action Scheduler %s', 'action-scheduler' ), $as_version ) . '</h2>' .
 					'<p>' .
 						__( 'Action Scheduler is a scalable, traceable job queue for background processing large sets of actions. Action Scheduler works by triggering an action hook to run at some time in the future. Scheduled actions can also be scheduled to run on a recurring schedule.', 'action-scheduler' ) .
+					'</p>' .
+					'<h3>' . __( 'WP CLI', 'action-scheduler' ) . '</h3>' .
+					'<p>' .
+						__( 'WP CLI commands are available: execute <code>wp help action-scheduler</code> for a list of available commands.' ) .
```

Comment:
We should escape strings whenever possible, avoid exposing translators to markup, and use text domains. Here's an alternative:

```suggestion
						sprintf(
							/* translators: %1$s contains the WP CLI command 'wp help action-scheduler' (not translatable). */
							esc_html__( 'WP CLI commands are available: execute %1$s for a list of available commands.', 'action-scheduler' ),
							'<code>wp help action-scheduler</code>'
						) .
```